bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClient.java

```diff
@@ -436,6 +443,86 @@ private TableInfo materializeTable(
     }
   }
 
+  public void loadDataIntoTable(
+      LoadDataOptions options,
+      List<String> sourceUris,
+      FormatOptions formatOptions,
+      JobInfo.WriteDisposition writeDisposition) {
+    Job finishedJob = null;
+    try {
```

Comment:
Can we reduce the scope of this try block? Not sure if we need it here since initially we are only setting up the objects.